src/org/usfirst/frc/team2335/robot/commands/Lift.java

```diff
@@ -0,0 +1,41 @@
+package org.usfirst.frc.team2335.robot.commands;
+
+import org.usfirst.frc.team2335.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class Lift extends Command
+{
+    public Lift() 
+    {
+        requires(Robot.climb);
+    }
+
+    
+    protected void initialize() 
+    {
+    	
+    }
+
+    protected void execute() 
+    {
+    	//Starts the climb process
+    	Robot.climb.startClimb();
+    }
+    
+    protected boolean isFinished() 
+    {
+        return false;
```

Comment:
In the operator interface file, it uses button.whileHeld, whileHeld automatically ends the command when the button is released, so this in here just means that we don't ever want the command to stop itself, but the scheduling from the operator interface file will stop the command.